tests/epics/test_signals.py

```diff
@@ -535,3 +536,26 @@ def test_signal_helpers():
 
     execute = epics_signal_x("Execute")
     assert execute._backend.write_pv == "Execute"
+
+
+async def test_str_enum_returns_enum(ioc: IOC):
+    await ioc.make_backend(MyEnum, "enum")
+    pv_name = f"{ioc.protocol}://{PV_PREFIX}:{ioc.protocol}:enum"
+
+    sig = epics_signal_rw(MyEnum, pv_name)
+    await sig.connect()
+    val = await sig.get_value()
+    assert val is MyEnum.b
```

Comment:
... but here the value `is` the Enum. How does that work?

Reply:
This is where datatype is an enum, so get_supported_values was map{str:enum} and the EnumConverter has the Enum values keyed by name (or index, appropriately)